.clang-format

```diff
@@ -7,7 +7,7 @@ AlignConsecutiveAssignments: None
 AlignConsecutiveBitFields: Consecutive
 AlignConsecutiveDeclarations: None
 AlignConsecutiveMacros: None
-AlignEscapedNewlines: DontAlign
+AlignEscapedNewlines: Right
```

Comment:
This makes side-by-side diffs impossible to read in code review. Can we keep the old behavior or use `Left` here instead?

Also, as a side note, please don't sneak in project-wide formatting changes into otherwise highly specific PRs.

Reply:
> This makes side-by-side diffs impossible to read in code review.

How so...?

> Can we keep the old behavior or use Left here instead?

Left would be a big improvement on the old behaviour, but Right is definitely less noisy (since the \'s then go away when reading code).

> Please don't sneak in project-wide formatting changes into otherwise highly specific PRs.

OK. This one seemed trivial but I guess not.

Reply:
> How so...?

The `Right` style apparently always pads up to the line character limit (140), so even on my MacBook with a 2560x1600 resolution, GitHub's side-by-side diff breaks each macro line into two.

> Left would be a big improvement on the old behaviour, but Right is definitely less noisy (since the 's then go away when reading code).

Arguably things for which `\` noise is a distinguishing factor shouldn't be macros in the first place.